src/mono_camera.cpp

```diff
@@ -162,4 +167,22 @@ void MonoCamera::updateCameraInfo(const avt_vimba_camera::AvtVimbaCameraConfig&
   info_man_->setCameraInfo(ci);
 }
 
+
+
+
+
+void MonoCamera::time_compare()
+{
+    ros::Duration img_time_difference=ros::Time::now()-last_time;
+    ros::Duration three_seconds(3.0);
+    if ((img_time_difference>three_seconds) && flag==cav_msgs::DriverStatus::OPERATIONAL)
+{
+    flag=cav_msgs::DriverStatus::OFF;
+}
+}
```

Comment:
I am think maybe we need to set timeout less than 3 seconds?
BTW, Can you clean up the blank lines and reformatting the indentation?